dali/operators/imgcodec/image_decoder.h

```diff
@@ -577,9 +577,10 @@ class ImageDecoder : public StatelessOperator<Backend> {
    * @brief Checks that nvImageCodec version is at least a given version
    */
   bool version_at_least(int req_major, int req_minor, int req_patch) {
-    int major = -1, minor = -1, patch = -1;
+    int req_majormajor = -1, minor = -1, patch = -1;
     get_nvimgcodec_version(&major, &minor, &patch);
-    return major >= req_major && minor >= req_minor && patch >= req_patch;
+    return MAKE_SEMANTIC_VERSION(major, minor, patch) >=
```

Comment:
I didn't know about this macro, is this something generic? :o

Reply:
Seems to nvimagecodec's https://github.com/NVIDIA/nvImageCodec/blob/bd192034b05991fb368913fe8afcc09f1711d399/include/nvimgcodec_version.h.in#L32